subsys/fs/shell.c

```diff
@@ -523,6 +533,13 @@ static int cmd_mount_fat(const struct shell *shell, size_t argc, char **argv)
 
 static int cmd_mount_littlefs(const struct shell *shell, size_t argc, char **argv)
 {
+#ifdef CONFIG_FS_LITTLEFS_BLK_DEV
+	static const char *disk_mount_pt = "/"CONFIG_SDMMC_VOLUME_NAME":";
+	static const char *disk_pdrv = CONFIG_SDMMC_VOLUME_NAME;
+
+	littlefs_mnt.storage_dev = (void *)disk_pdrv;
+	littlefs_mnt.mnt_point = disk_mount_pt;
```

Comment:
(note) if you set this parameter here you will never execute below lines, see further check
moreover, looks like we get mount point from shell: 
```
	char *mntpt = mntpt_prepare(argv[1]);
	if (!mntpt) {
		shell_error(shell, "Failed to allocate mount point");
		return -ENOEXEC; /* ?!? */
	}
	littlefs_mnt.mnt_point = mntpt;
```

Reply:
still not fixed or at least answer why shouldn't you fix it ?